addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3011,6 +3012,37 @@ export class OdooEditor extends EventTarget {
                         );
                     }
                 }
+                if (ev.data === '`') {
+                    // We just inserted a backtick, check if there was another
+                    // one before.
+                    const range = getDeepRange(this.editable, { joinText: true, select: true });
+                    const textHasOpeningTick = /`.*`/.test(range.startContainer.textContent);
```

Comment:
What about is you have like an image in the middle or something else entirely ? I think that's going to match regardless.

Reply:
No because we got a deep range so the startContainer is a text node.